lib/rucio/web/rest/flaskapi/v1/rules.py

```diff
@@ -150,14 +154,16 @@ def put(self, rule_id):
             description: No rule found for the given id
        """
         parameters = json_parameters()
-        options = param_get(parameters, 'options')
+        options: Dict[str, Any] = param_get(parameters, 'options', default={})
+        print(options)
```

Comment:
Ditto.

Reply:
i added type hints to the options dict and a default value, i dont think that that has to go. on the other hand, i will remove the `print` statement.